gadgetchains/Laminas/RCE/1/chain.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace GadgetChain\Laminas;
+
+class RCE1 extends \PHPGGC\GadgetChain\RCE\FunctionCall
+{
+    public static $version = '2.1.5 <= 2.5.3 & 2.11.2';
```

Comment:
Which package(s) are these versions for?

Reply:
it's https://github.com/laminas/laminas-http